contracts/test-helpers/ValidatorTestWrapper.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.16;
+
+import {ICircuitValidator} from "../interfaces/ICircuitValidator.sol";
+
+contract ValidatorTestWrapper {
```

Comment:
VerifierTestWrapper ?

Reply:
No, this is exactly Validator test wrapper but not Verifier. It's main purpose is to add msg.sender inside wrapper to the calldata before calling the validator.